src/main/preload/dropdown.js

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) 2016-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+'use strict';
+
+import {ipcRenderer} from 'electron';
+
+import {
+    UPDATE_TEAMS_DROPDOWN,
+    REQUEST_TEAMS_DROPDOWN_INFO,
+    RECEIVE_DROPDOWN_MENU_SIZE,
+    SEND_DROPDOWN_MENU_SIZE,
+    SWITCH_SERVER,
+    CLOSE_TEAMS_DROPDOWN,
+    SHOW_NEW_SERVER_MODAL,
+} from 'common/communication';
+
+console.log('preloaded for the dropdown!');
```

Comment:
We usually keep these here in case we need them, just to make sure the preload script has loaded. The dev tools isn't accessible by the end user for this browser view so the user will never see it. I suppose we can remove it if need be.